app/views/whatsrunningwhere/DeploymentTimeline.scala.html

```diff
@@ -88,10 +89,9 @@
                 return `<div class="timeline-tooltip"><b> ${ver} deployed by ${user} </b><br>From ${start} To  ${end}<br><em>Deployed for ${days} days</em> </div>`
             }
     </script>
-    <h2>Deployment Timeline @if(service != ""){ - @service}</h2>
+   <h2>Deployment Timeline @if(service != ""){<a id="link-to-service-info-page" href="@CatalogueController.repository(service)">- @service</a></h2>}
```

Comment:
I know it's not part of your change, but I wonder why we guard against the empty string here? If it's expected that this page could render without a given service, an `Option[String]` would be a better representation.

Either way, `CatalogueController.service` might make more sense than the generic `repository` endpoint too.